Gemfile

```diff
@@ -100,3 +100,5 @@ group :test, :development do
   gem "rspec"
   gem "rspec-rails"
 end
+
+gem "redcarpet", "~> 3.6"
```

Comment:
Not a blocker, but could you move this higher up the Gemfile?
Just so there is a grouping of "Gems used in all environments", "Gems used in Development", "Gems used in Test", etc.